# A custom controller for test
test_controller:
    type: cobot_controllers/CartesianPoseController
    arm_id: panda

velocity_controller:
    type: cobot_controllers/CartesianVelocityController
    arm_id: panda
